packages/utils/src/utils/split-utxos.ts

```diff
@@ -0,0 +1,23 @@
+import { ErrorCode, FuelError } from '@fuel-ts/errors';
+import type { AbstractAddress } from '@fuel-ts/interfaces';
+import type { BN } from '@fuel-ts/math';
+
+export function splitUTXOs(
+  balance: BN,
+  amount: BN,
+  assetId: string,
+  destination: AbstractAddress,
```

Comment:
Maybe `destination` could just take a string as it is returning a string? And then we don't need the unknown cast in the above test?

Reply:
In my opinion the benefits of enforcing the address type outweigh the cast in the test. The primary goal here is to validate user input not to have "clean" tests.

Reply:
No longer relevant 7165a1a85